packages/panzer/core/cmake/PanzerCore_config.hpp.in

```diff
@@ -8,7 +8,7 @@
 #cmakedefine HAVE_MPI
 
 // What AD Type are you going to use?
-#cmakedefine PANZER_FADTYPE @Panzer_ENABLE_FADTYPE@
+#cmakedefine PANZER_FADTYPE @PANZER_FADTYPE@
```

Comment:
@rppawlo, I think this change is actually correct, no?  I think you want to use the CMake cache var ` PANZER_FADTYPE` for this instead of `Panzer_ENABLE_FADTYPE`?  (But I guess in this case it does not matter.)

Reply:
ok. sounds good.